test/DynamoCoreTests/Extensions/ExtensionCommandExecutiveTest.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Dynamo.Extensions;
+using Dynamo.Models;
+using Dynamo.Scheduler;
+using Moq;
+using NUnit.Framework;
+
+namespace Dynamo.Tests.Extensions
+{
+    [TestFixture]
+    class ExtensionCommandExecutiveTest
+    {
+        private int cmdExecutionState = -1;
+        private ICommandExecutive executive;
+        private Mock<IExtension> extMock;
+        private Logging.NotificationMessage message;
+        private DynamoModel model;
+
+        //This callback method will be used in the Mocked Extension
+        private void ExtensionReadyCallback(ReadyParams ready)
+        {
+            executive = ready.CommandExecutive;
+            Assert.IsTrue(ready.WorkspaceModels.Any());
+            Assert.IsNotNull(ready.CurrentWorkspaceModel);
+            ready.NotificationRecieved += (Logging.NotificationMessage obj) => message = obj;
```

Comment:
Would it be unsubscribed itself or we still need to clean it up?

Reply:
@QilongTang I've removed the line where is subscribing to the event (not used) also deleted the a field since is not used.
commit: https://github.com/DynamoDS/Dynamo/pull/10740/commits/0838d8b50d278a703dd5bc1cd4cbceba54eae64f